scripts/docker_cmake

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+rm -f  CMakeCache.txt
+rm -rf CMakeFiles/
```

Comment:
@dalg24 wrote:

> If we remove these guys we will have to recompile everything every time we reconfigure.

True. But if I understand correctly, this would only happen when you use `docker_cmake`, aka when you build using docker container. That's what I was doing and at some point I was hit by some internal CMake inconsistencies that resulted in faulty build.